### ISSUE TITLE
MySQL support

### ISSUE THREAD

**Issue by popovserhii** [goldcaddy77/warthog#96]
README contains the next declaration:
> You must have Postgresql installed to use Warthog. 

Do you plan to support other RDB which are supported by [TypeORM](https://typeorm.io)?

**Comment by goldcaddy77** [comment]
Yes, this should be relatively easy to do. Feel free to take a crack at it if you'd like and shoot in a PR. The items you'll need to cover are:

- mapping of DB data types to driver in TypeORMConverter
- ensuring json and jsonb support continues to work
- you'll need to come up with your own ways to create and drop DBs and won't have docker support out of the box from [warthog-starter](https://github.com/goldcaddy77/warthog-starter)
